docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -165,39 +169,42 @@ modelConfig {
 }
 ```
 
+&nbsp;
 ### Configuring replicas count
 
-Each scenario has its own, configured parallelism. It describes how many worker threads,
```

Comment:
Decideded to rewrite this section as the fragment on replicas count in R-R sounded like simply bolted onto the previous content without thinking about the overall structure. 

Also, decided to remove terms  _worker thread_ and _workers_ . The setting is called _tasksPerReplica_, so referring to it later as to _worker threads_ is misleading. Is it something different from these tasks?